src/main/java/team/baymax/logic/commands/patient/EditPatientCommand.java

```diff
@@ -29,7 +29,7 @@
  */
 public class EditPatientCommand extends Command {
 
-    public static final String COMMAND_WORD = "editpatient";
+    public static final String COMMAND_WORD = "editpat";
```

Comment:
shotening to "pat" from "patient" could be confusing for the user as "pat" is not a conventional short form for patient (unlike appt)